deployment-templates/.goreleaser.yml

```diff
@@ -3,9 +3,9 @@ archives:
     name_template: '{{ .Binary }}-{{ .Tag }}-{{ .Os }}-{{ .Arch }}'
 before:
   hooks:
-    - make build_provider
+    - make tfgen
 builds:
-  - binary: provider-xyz
+  - binary: provider-resource-xyz
```

Comment:
@jkisk just bumped into an issue with @rawkode. The binary name needs to be:

`pulumi-resource-xyz`

not 

`provider-resource-xyz`

Can you fix this?

Reply:
Feel free to send a PR @ringods - will happily merge it :)

Reply:
@stack72 https://github.com/pulumi/pulumi-tf-provider-boilerplate/pull/83